src/contrib/mlir/CMakeLists.txt

```
@@ -77,10 +77,10 @@ function(whole_archive_link target)
     set_target_properties(${target} PROPERTIES LINK_FLAGS ${link_flags})
 endfunction(whole_archive_link)
 
-whole_archive_link(mlir_backend
+set(LIBS
     ${LLVM_BUILD_LIBRARY_DIR}/libMLIRAffineOps.a
-    ${LLVM_BUILD_LIBRARY_DIR}/libMLIRStandardOps.a
-)
+    ${LLVM_BUILD_LIBRARY_DIR}/libMLIRStandardOps.a)
```

Just move brace on new line to be consistent with rest of file
```suggestion
    ${LLVM_BUILD_LIBRARY_DIR}/libMLIRStandardOps.a
)
```